tft_bot/resources/config.yaml

```diff
@@ -9,6 +9,20 @@ verbose: false
 # The CLI flag -f/--ffearly will override this.
 forfeit_early: false
 
+# The traits you want the bot to roll for.
+# The trait names are almost as written in-game, with spaces replaced by _ and . and : being ignored.
+# For example: admin, star_guardian, mecha_prime, lasercorps
```

Comment:
Would it be worth either mentioning the folder which these 'possible traits' can be found, pointing to the config array, or just straight up listing the possible 'valid' options? 🤔

Reply:
Been pondering this as well; I think dropping a link to `https://github.com/Kyrluckechuck/TFT-Bot/tree/main/captures/trait` is probably the best. Will update the comment :)